course-v2/assets/course-v2.ts

```diff
@@ -26,13 +34,14 @@ $(function() {
   initCourseDrawersClosingViaSwiping()
 })
 
-// @ts-expect-error for window.initVideoJS()
+window.videoJSLoaded = false
+window.nanogallery2Loaded = false
```

Comment:
I don't think there's any reason to put these on the global `window` object. We could scope them:

```ts
let videoJSLoaded = false
window.initVideoJS = () => {
  if (videoJSLoaded) return
  initVideoDownloadPopup()
  import("./videojs-imports").then(module => {
    module.initVideoJS()
  })
}
```

Or we could create a helper version of `once` without lodash:
```ts
const once = (callback: () => void) => {
  let hasRun = false
  if (!hasRun) {
    callback()
  }
}
```

Either way, I think it's worth putting fewer things on the global object.